tests/pkg_elk/main.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2021 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the Elk Javascript engine
+ *
+ *              See README at https://github.com/cesanta/elk
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+#include <stdio.h>
+#include "elk.h"
+
+/* C function that adds two numbers. Will be called from JS */
+int sum(int a, int b) {
```

Comment:
I'd give that a more unique name so it's clearer where this is coming from

```suggestion
int c_sum_from_riot(int a, int b) {
```

Reply:
This application is copied as-is (well, comments were converted to C-style comments) from the project README.

Reply:
Simple as it might be, it's probably GPLv2 . :/
Maybe contact the author to clarify?

(Assuming Inria copyright is probably not correct.)

Reply:
@kaspar030, good point about GPL, I'll ask. Regarding the copyright, it was automatically generated using riotgen ;) but still better clarify this also (since it's indeed not Inria copyright).

Reply:
@cpq, are you ok with integrating Elk in RIOT as a package and also to use this application as an example (the application is taken from the README), if I change the copyright to "Cesanta Software Limited" ?

Reply:
@aabadie Absolutely. Please keep the copyright. Elk is dual licensed - GPLv2 and commercial. Commercial projects that would like to avoid GPLv2, can get a commercial license from us that does not have GPLv2 restrictions.